huxley/www/static/js/huxley/components/RegistrationView.js

```diff
@@ -89,6 +87,9 @@ var RegistrationView = React.createClass({
     CountryStore.getCountries(function(countries) {
       this.setState({countries: countries});
     }.bind(this));
+    CommitteeStore.getSpecialCommittees(function(committees) {
+      this.setState({committees: committees});
+    }.bind(this));
   },
```

Comment:
Variable choice here seems a bit confusing. Maybe we should consider refactoring both the arguments of the inner functions to be a little more clear as to what is going on in this function.